<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-C6RI43K" name="Default test" tests="1" failures="0" timestamp="29 Feb 2020 17:11:42 GMT" time="0.015" errors="0">
  <testcase name="@BeforeClass beforeClass" time="0.0" classname="pom.testcases.TC004">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at utils.Reporter.startTest(Reporter.java:20)
	at wrappers.ProjectWrappers.beforeClass(ProjectWrappers.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeClass -->
  <testcase name="@BeforeMethod irctcSignup" time="0.0" classname="pom.testcases.TC004">
    <skipped/>
  </testcase> <!-- @BeforeMethod irctcSignup -->
  <testcase name="@AfterMethod closeIrctc" time="0.0" classname="pom.testcases.TC004">
    <skipped/>
  </testcase> <!-- @AfterMethod closeIrctc -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="pom.testcases.TC004">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="pom.testcases.TC004">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="tc004" time="0.0" classname="pom.testcases.TC004">
    <skipped/>
  </testcase> <!-- tc004 -->
</testsuite> <!-- Default test -->
